### PR TITLE
jenv: add Chinese translation

### PR DESCRIPTION
**Pull request by BillLucky**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/common/jenv.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`jenv global {{java_version}}`
+
+- 设置当前 shell 会话的 JDK 版本：
+
+`jenv shell {{java_version}}`
+
+- 启用 jEnv 插件：
+
+`jenv enable-plugin {{plugin_name}}`
```
Comment (navarroaxel) [inline pages.zh/common/jenv.md:28]: Please translate the arguments for get this easier for non-English speakers, like `plugin_name` and `path/to/jdk_home`.

File: pages.zh/common/jenv.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# jenv
+
+> 管理”JAVA_HOME“环境变量的命令行工具。
+> 更多信息： <https://www.jenv.be/>.
+
+- 向 jEnv 添加一个 java 版本：
```
Comment (marchersimon) [inline pages.zh/common/jenv.md:6]:
```suggestion
- 向 jEnv 添加一个 Java 版本：
```

File: pages.zh/common/jenv.md (hunk 1 above)
Comment (BillLucky) [reply pages.zh/common/jenv.md]: OK.

**Review by marchersimon — APPROVED** [review]
Thanks @BillLucky

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**